### PR TITLE
AnythingElse.txt

### PR DESCRIPTION
**Pull request by mwhapeles**

### CODE REVIEW

**Comment by mwhapeles** [comment]
offended

File: AnythingElse.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+Thank You for helping <3
```
Comment (A-Tanner) [inline AnythingElse.txt:1]: Very cool message!

**Review by A-Tanner — COMMENTED** [review]
Love this!

**Comment by mwhapeles** [comment]
asreaew

**Comment by mwhapeles** [comment]
offended pt.2

**Comment by miklast** [comment]
Ray good

**Pull request merged**